tests/stress/tx_stress/test_stress_txs.py

```diff
@@ -0,0 +1,58 @@
+from pytest_testconfig import config as testconfig
+import time
+import multiprocessing as mp
+
+from tests.convenience import sleep_print_backwards
+from tests.queries import get_release_tick_msgs
+from tests.test_bs import setup_network, add_curl, setup_bootstrap, start_poet, setup_clients, wait_genesis
+from tests.tx_generator import actions
+from tests.tx_generator import config as conf
+from tests.tx_generator.models.wallet_api import WalletAPI
+from tests.tx_generator.models.accountant import Accountant
+
+
+def wait_for_next_layer(namespace, timeout):
+    tts = 5
+    old_release_ticks = get_release_tick_msgs(namespace, namespace)
+    time.sleep(tts)
+    new_release_tics = get_release_tick_msgs(namespace, namespace)
+
+    while len(old_release_ticks) == len(new_release_tics) and timeout > 0:
```

Comment:
note that here you are waiting for the first node that will release the next tick, and not all nodes. it's ok if this is the expected behaviour, also, I suspect that in case we have log jitter (meaning not all logs came at the same time, but a new layer havn't started yet, this method will not work properly, 
Perhaps you need to know exactly how many release tick messages you are expecting, something like `old_release_ticks % number_of_clients == 0`

Reply:
fixed in 24db1a1e4a26890048ad8a113991652ed0ed3ac3